context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -182,6 +182,9 @@ def get_conf_cells(self):
                 is_mock=self._is_mock,
                 base_name=region,
                 imaging_path=STITCHED_IMAGE_DIR,
+                mask_path=STITCHED_IMAGE_DIR.replace('expressions', 'mask'),
```

Comment:
This is probably paranoid, but would it be good to make this a tighter match? ... or maybe that would introduce fragility you don't want...

Reply:
I think this is OK for now. I see what you mean here but I think this OK especially since this may have to be reused for non Cytokit data.